BACKENDS.md

```diff
@@ -77,6 +87,66 @@ On various platforms, the clock can get out of sync between the host machine and
 This is particularly problematic if using `check_time: true` in the DC/OS configuration.
 To work around this, run `docker run --rm --privileged alpine hwclock -s`.
 
+## `dcos_e2e.backend.AWS`
+
+```python
+AWS(
+    aws_access_key_id,
+    aws_secret_access_key,
+    aws_region,
+    admin_location,
+    instance_type,
+    deploy_timeout=60,
+    workspace_dir=None,
+)
+```
+
+### Parameters
+
+#### `aws_access_key_id`
+
+The access key ID which must be specified as part of the authentication with AWS.
+
+#### `aws_secret_access_key`
+
+The secret access key which must be specified as part of the authentication with AWS.
+
+#### `aws_region`
+
+The AWS region that the backend should create the clusters in. This must be a string representation of an [AWS region code](http://docs.aws.amazon.com/general/latest/gr/rande.html). The following regions are supported.
+
+* `'ap-northeast-1'`
+* `'ap-southeast-1'`
+* `'ap-southeast-2'`
+* `'eu-central-1'`
+* `'eu-west-1'`
+* `'sa-east-1'`
+* `'us-east-1'`
+* `'us-west-1'`
+* `'us-west-2'`
+
+#### `admin_location`
+
+IP address range of which the corresponding hosts are allowed to access nodes of clusters created by this backend. Specifiying ``'0.0.0.0/32'`` will allow anyone to connect.
+
+#### `instance_type`
+
+The [AWS instance type](https://aws.amazon.com/ec2/instance-types/) for all hosts of clusters created by this backend.
+
+#### `deploy_timeout`
+
+The lifespan measured in minutes for clusters created by this backend.
+
+#### `workspace_dir`
```

Comment:
In the Docker backend, this is required because we make some really big files. Is this necessary for here? Maybe remove this and only add it if we need it.

Reply:
It's useful to find the generated SSH key if you want to retrieve it for some purpose.

Reply:
FWIW I've changed my perspective and in the `Docker` backend, having a workspace is now really useful - we can clean it up as part of destroying a cluster with the CLI.